docs/api/toolkits.md

```diff
@@ -0,0 +1,112 @@
+# Toolkits Wrappers and Registries
+
+The toolkit wrappers provide a simple uniform API for accessing minimal functionality of cheminformatics toolkits.
+
+These toolkit wrappers are generally used through a {class}`ToolkitRegistry`, which can be constructed with a desired precedence of toolkits:
+
+```python
+>>> from openff.toolkit.utils.toolkits import ToolkitRegistry, OpenEyeToolkitWrapper, RDKitToolkitWrapper, AmberToolsToolkitWrapper
+>>> toolkit_registry = ToolkitRegistry()
+>>> toolkit_precedence = [OpenEyeToolkitWrapper, RDKitToolkitWrapper, AmberToolsToolkitWrapper]
+>>> [ toolkit_registry.register_toolkit(toolkit) for toolkit in toolkit_precedence if toolkit.is_available() ]
+[None, None, None]
```

Comment:
Why not

```python
toolkit_registry = ToolkitRegistry(toolkit_precedence = [OpenEyeToolkitWrapper, RDKitToolkitWrapper, AmberToolsToolkitWrapper])
```

This block implies to me that users are responsible for registering their own toolkits. I don't know if this is the case, and I assume/hope that's automatically handled when you make a `ToolkitRegistry` in one go.

I might be useful to append

```
>>> toolkit_registry
<ToolkitRegistry containing OpenEye Toolkit, The RDKit, AmberTools>
```
just to demonstrate what's in the thing